keyboards/handwired/dactyl_manuform_pi_pico/keyboard.json

```diff
@@ -0,0 +1,127 @@
+{
+    "manufacturer": "GustawXYZ",
+    "keyboard_name": "dactyl_manuform_pi_pico",
+    "maintainer": "GustawXYZ",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "url": "https://github.com/GustawXYZ/dactyl_manuform_pi_pico",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "split": {
+        "enabled": true,
+        "handedness": {
+                "pin": "GP22"
+            },
+            "serial": {
+                "driver": "vendor"
+            },
+            "transport": {
+                "watchdog": true
+            }
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["GP10", "GP11", "GP12", "GP13", "GP14", "GP15"],
+        "rows": ["GP21", "GP20", "GP19", "GP18", "GP17", "GP16"]
+    },
+    "ws2812": {
+        "pin": "GP22",
+        "driver": "vendor"
+    },
```

Comment:
WS2812 configuration but no RGB Light or Matrix configuration? If not, this should be removed.

Reply:
Thanks for the review. Removed.